framework/helpers/BaseHtml.php

```diff
@@ -975,16 +980,14 @@ public static function radioList($name, $selection = null, $items = [], $options
             }
             $index++;
         }
+        $visibleContent = implode($separator, $lines);
 
-        if (isset($options['unselect'])) {
-            // add a hidden field so that if the list box has no option being selected, it still submits a value
-            $hidden = static::hiddenInput($name, $options['unselect']);
-            unset($options['unselect']);
-        } else {
-            $hidden = '';
+        if (false === $tag || 'false' === $tag) {
```

Comment:
We usually write the variable first: `$tag === false`

Also I don't think that `fasle` as a string should do the same as a boolean `false`.